xml2kinto/__main__.py

```diff
@@ -1,10 +1,12 @@
 import os
+import argparse
+
 from xml2kinto.synchronize import synchronize
 
 # options to move to a config file
 xml_file = os.path.abspath(os.path.join(os.path.dirname(__file__),
                                         '..', 'blocklist.xml'))
-auth = ('mark', 'p4ssw0rd')
```

Comment:
Please do not change that, it would let use pass a Authorization backend to requests.